### PR TITLE
Pods Service - Listing, Creation, Deletion

### PR DESCRIPTION
**Pull request by NotChristianGarcia**
## Related Github Issues:

- [TUI-354](https://github.com/tapis-project/tapis-ui/issues/354)

## Summary of Changes:
Added Pods to sidebar. Adding listing, creation, and deleting of pods.

## Testing Steps:

1. Run `npm run test`. 
  - One test for nav.
  - There was a migrated `podsListing` feature with tests, that's not currently used, so skipped.

## UI Photos:
Screenshot from 2024-03-14 12-46-44

## Notes:
yay

### CODE REVIEW

File: src/fixtures/pods.fixtures.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { Pods } from '@tapis/tapis-typescript';
```
Comment (nathandf) [inline src/fixtures/pods.fixtures.ts]: Remove unused imports

File: src/tapis-app/Pods/_components/PodToolbar/CreatePodModal/Settings/AdvancedSettings.tsx
Comment (nathandf) [inline src/tapis-app/Pods/_components/PodToolbar/CreatePodModal/Settings/AdvancedSettings.tsx:1]: Looks like you used Systems components as a template but forgot to remove the Systems stuff

File: src/tapis-app/Pods/_components/PodToolbar/CreatePodModal/Settings/Batch/BatchLogicalQueuesSettings.tsx
Comment (nathandf) [inline src/tapis-app/Pods/_components/PodToolbar/CreatePodModal/Settings/Batch/BatchLogicalQueuesSettings.tsx:1]: More Systems!

File: src/tapis-app/Pods/_components/PodToolbar/CreatePodModal/Settings/Batch/BatchSettings.tsx
Comment (nathandf) [inline src/tapis-app/Pods/_components/PodToolbar/CreatePodModal/Settings/Batch/BatchSettings.tsx:1]: You get a system! and you get a system!

File: src/tapis-app/Pods/_components/PodToolbar/CreatePodModal/Settings/CmdSettings.tsx
Comment (nathandf) [inline src/tapis-app/Pods/_components/PodToolbar/CreatePodModal/Settings/CmdSettings.tsx:1]: System

File: src/tapis-app/Pods/_components/PodToolbar/CreatePodModal/Settings/DtnSettings.tsx
Comment (nathandf) [inline src/tapis-app/Pods/_components/PodToolbar/CreatePodModal/Settings/DtnSettings.tsx:1]: More Systems

File: src/tapis-app/Pods/_components/PodToolbar/CreatePodModal/Settings/Job/JobCapabilitiesSettings.tsx
Comment (nathandf) [inline src/tapis-app/Pods/_components/PodToolbar/CreatePodModal/Settings/Job/JobCapabilitiesSettings.tsx:1]: More Systems

File: src/tapis-ui/components/pods/PodListing/PodListing.module.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.system-list {
```
Comment (nathandf) [inline src/tapis-ui/components/pods/PodListing/PodListing.module.scss:1]: Still referencing systems, was this an accident?

**Review by nathandf — CHANGES_REQUESTED** [review]

File: src/tapis-ui/components/pods/PodListing/PodListing.module.scss (hunk 2 above)
Comment (NotChristianGarcia) [reply src/tapis-ui/components/pods/PodListing/PodListing.module.scss:1]: It's vestigial, might use it, but I'll delete it as it's in Git's history now.

**Review by nathandf — APPROVED** [review]

**Pull request merged**